components/Releases/ReleaseLayout.jsx

```diff
@@ -20,67 +20,103 @@ export default function ReleaseLayout({
   const [releaseDate, setReleaseDate] = useState(new Date(release.release_date))
+  const [isClient, setIsClient] = useState(false)
 
   const sanitizedAbout = sanitize(release.about)
 
+  useEffect(() => {
+    setIsClient(true)
+
+    // if (
+    //   release.player_embed &&
+    //   release.player_embed.includes("album=") &&
+    //   !albumCode
+    // ) {
+    //   let embedArray = release.player_embed.split("/")
+
+    //   embedArray.forEach((value) => {
+    //     if (value.match(/(?:album=)\d+/)) {
+    //       setAlbumCode(value.slice(6))
+    //     }
+    //   })
+    // }
+  }, [])
+
   return (
     <>
       <SEO
         title={`${release.title} by ${release.artist}`}
         description={`Get a download code for ${release.title}`}
       ></SEO>
-      <div className={cn(styles.wrapper, "stack inline-max")}>
-        <Image
-          className={styles.artwork}
-          src={release.artwork_url || "/default-image-release.png"}
-          alt={release.title}
-          height={250}
-          width={250}
-        />
-        <div>
-          <h1 className={styles.title}>{release.title}</h1>
-          {userType == "artist" ? (
-            <Link href={`/${userSlug}`} className={styles.artist}>
-              {release.artist}
-            </Link>
-          ) : (
-            <p className={styles.artist}>{release.artist}</p>
-          )}
-          {userType == "label" ? (
-            <Link href={`/${userSlug}`} className={styles.label}>
-              {release.label}
-            </Link>
-          ) : (
-            <p className={styles.label}>{release.label}</p>
-          )}
-          <p>
-            {release.type == "Choose release type" ? null : release.type}{" "}
-            {release.release_date ? `  — ${releaseDate.getFullYear()}` : null}
-          </p>
-        </div>
-        {!authorized ? (
-          <InputPagePassword
-            setAuthorized={setAuthorized}
-            pagePassword={release.page_password}
-            label="Enter password to generate Bandcamp code"
+
+      {isClient ? (
+        <div className={cn(styles.wrapper, "stack inline-max")}>
+          <Image
+            className={styles.artwork}
+            src={release.artwork_url || "/default-image-release.png"}
+            alt={release.title}
+            height={250}
+            width={250}
           />
-        ) : (
-          <div className={styles.codes}>
-            <CodeGenerator release={release} profileYumLink={profileYumLink} />
+          {release.player_embed.length > 0 ? (
+            <section>
+              <iframe
+                style={{ border: 0, width: 100 + "%", height: 120 + "px" }}
+                src={`https://bandcamp.com/EmbeddedPlayer/album=${release.player_embed}/size=large/bgcol=ffffff/linkcol=0687f5/tracklist=false/artwork=small/transparent=true/`}
+                seamless
+              >
+                <a href=""></a>
```

Comment:
Render the track url and text from the embed code. If the iframe fails to load then this shows up in its place. For example:

```
<a href="https://frank.bandcamp.com/album/frankie-friday">Frankie Friday by Frank</a>
```